sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb

```diff
@@ -0,0 +1,1094 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # for i in range(len(df)):
</pre><p>Remove commented lines? There's several for-statements and df.head() statements commented out throughout the notebook. Preferred to either remove or uncomment the lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3016/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Sure.. removed these commented lines</p>